analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/InitializationVectorShouldBeRandomTest.cs

```diff
@@ -19,7 +19,7 @@
  */
 
 using SonarAnalyzer.Rules.CSharp;
-using SonarAnalyzer.Rules.SymbolicExecution;
+using SonarAnalyzer.SymbolicExecution.Sonar.Analyzers;
```

Comment:
Line 24: Let's align the namespace to `SonarAnalyzer.UnitTest.Rules` like most of the rules.

There's not practical reason of moving these to a separate NS.